_IBC/sei-nois.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
```

Comment:
Please use alphabetical order to name the file and to determine which chain is chain1 and which is chain2. This needs to be reversed

Reply:
Done!